modules/Bio/EnsEMBL/Compara/PipeConfig/EG/ProteinTrees_conf.pm

```diff
@@ -205,43 +149,16 @@ sub default_options {
         # Add the database location of the previous Compara release. Use "undef" if running the pipeline without reuse
         'prev_rel_db' => undef,
 
-    # Configuration of the pipeline worklow
-
-        # How will the pipeline create clusters (families) ?
-        # Possible values: 'blastp' (default), 'hmm', 'hybrid'
-        #   'blastp' means that the pipeline will run a all-vs-all blastp comparison of the proteins and run hcluster to create clusters. This can take a *lot* of compute
-        #   'hmm' means that the pipeline will run an HMM classification
-        #   'hybrid' is like "hmm" except that the unclustered proteins go to a all-vs-all blastp + hcluster stage
-        #   'topup' means that the HMM classification is reused from prev_rel_db, and topped-up with the updated / new species  >> UNIMPLEMENTED <<
-        'clustering_mode'           => 'hybrid',
-
-        # How much the pipeline will try to reuse from "prev_rel_db"
-        # Possible values: 'clusters' (default), 'blastp', 'members'
-        #   'members' means that only the members are copied over, and the rest will be re-computed
-        #   'hmms' is like 'members', but also copies the HMM profiles. It requires that the clustering mode is not 'blastp'  >> UNIMPLEMENTED <<
-        #   'hmm_hits' is like 'hmms', but also copies the HMM hits  >> UNIMPLEMENTED <<
-        #   'blastp' is like 'members', but also copies the blastp hits. It requires that the clustering mode is 'blastp'
-        #   'clusters' is like 'hmm_hits' or 'blastp' (depending on the clustering mode), but also copies the clusters
-        #   'alignments' is like 'clusters', but also copies the alignments  >> UNIMPLEMENTED <<
-        #   'trees' is like 'alignments', but also copies the trees  >> UNIMPLEMENTED <<
-        #   'homologies is like 'trees', but also copies the homologies  >> UNIMPLEMENTED <<
-
     # CAFE parameters
 
     # GOC parameters
         'goc_taxlevels'             => [],  # this is the default default
         'goc_taxlevels_fungi'       => [],
         'goc_taxlevels_metazoa'     => ['Diptera', 'Hymenoptera', 'Nematoda'],
-        'goc_taxlevels_plants'      => ['solanum', 'fabids', 'Brassicaceae', 'Pooideae', 'Oryzoideae', 'Panicoideae'],
         'goc_taxlevels_protists'    => [],
         'goc_taxlevels_vb'          => ['Chelicerata', 'Diptera', 'Hemiptera'],
 
     # Extra analyses
-        # Do we want the Gene QC part to run ?
-        'do_gene_qc'                    => 0,
-        # Do we extract overall statistics for each pair of species ?
-        'do_homology_stats'             => 0,
```

Comment:
I can't see where these are getting set now?

Reply:
They are in the parent class `PipeConfig/ProteinTrees_conf.pm`, lines 351 and 353.

Reply:
I am blind. Need new glasses.